### DISCUSSION TITLE
Welcome to react-native-tts Discussions!

### DISCUSSION THREAD

**Discussion by ak1394** [ak1394/react-native-tts#152] [General]
Until now the only way to discuss react-native-tts related questions was to use Github Issues. Now with Discussions available on the repository there is a better place to have these conversations.

Please use Github Discussions to talk about anything related to use of react-native-tts, which is not a bug in react-native-tts itself.

**Comment by Adityapassi** [comment]
Does react-native-tts provide any method which returns mp3/audio file

**Reply by ak1394** [reply]
Not in the currently released version. There is some unfinished work to write 'wav' files on 'write_to_file' branch
